### DISCUSSION TITLE
Bakaxl上面的联机什么时候能够应用到pcl上呢？

### DISCUSSION THREAD

**Discussion by ghost** [Meloong-Git/PCL#2795] [讨论]
Bakaxl上面的联机已经开始测试了，非常期待pcl早点加入这个功能

**Comment by DreamUniverse843** [comment]
#2788 #2794
等龙猫跟进叭

**Comment by Pigeon0v0** [comment]
![image](https://github.com/Hex-Dragon/PCL2/assets/60414767/a7d05826-3d6e-4a4a-a9be-5045d162242a)

**Reply by HomoobyDX** [reply]
目前bakaxl的联机还是测试，所以不会很快上到PCL和HMCL
